libmscore/cmd.cpp

```diff
@@ -1883,15 +1883,6 @@ void Score::cmdResetBeamMode()
             deselectAll();
       }
 
-//---------------------------------------------------------
```

Comment:
This should probably be in the 2nd PR, along with the editstyle.ui changes.  I'll work on that...

Reply:
oops, forgot to remove a comment line, next push it will be fixed.